packages/core/src/application_ref.ts

```diff
@@ -1169,3 +1244,30 @@ export function provideNgZoneChangeDetection(ngZone: NgZone): StaticProvider[] {
     {provide: ZONE_IS_STABLE_OBSERVABLE, useFactory: isStableFactory},
   ];
 }
+
+/**
+ * Provides `NgZone`-based change detection for the application.
```

Comment:
```suggestion
 * Provides `NgZone`-based change detection for the application bootstrapped using
 * `bootstrapApplication` method.
```
(may be also add a note that it's not supported in `platformBrowser().bootstrapModule`?)